stable/owncloud/README.md

```diff
@@ -51,6 +51,10 @@ The following tables lists the configurable parameters of the ownCloud chart and
 | `imagePullPolicy`                   | Image pull policy                         | `IfNotPresent`                              |
 | `ingress.enabled`                   | Enable use of ingress controllers         | `false`                                     |
 | `ingress.servicePort`               | Ingress' backend servicePort              | `http`                                      |
+| `ingress.annotations`               | An array of service annotations           | `nil`                                       |
+| `ingress.tls[i].secretName          | The secret kubernetes.io/tls              | `nil`                                       |
```

Comment:
I think this is confusing. Just add one line for `ingress.tls`.

Reply:
Fixed